docs/deploy.md

```diff
@@ -1,38 +1,66 @@
 # Deploy models with Cog
 
-Cog containers are Docker containers that serve an HTTP server for running predictions on your model. You can deploy them anywhere that Docker containers run.
+Cog containers are Docker containers that serve an HTTP server 
+for running predictions on your model. 
+You can deploy them anywhere that Docker containers run.
 
-This guide assumes you have a model packaged with Cog. If you don't, [follow our getting started guide](getting-started-own-model.md), or use [an example model](https://github.com/replicate/cog-examples).
+This guide assumes you have a model packaged with Cog. 
+If you don't, [follow our getting started guide](getting-started-own-model.md), 
+or use [an example model](https://github.com/replicate/cog-examples).
 
 ## Getting started
 
 First, build your model:
 
-    cog build -t my-model
+```console
+cog build -t my-model
+```
 
 Then, start the Docker container:
 
-    docker run -d -p 5000:5000 my-model
+```shell
+# If your model uses a CPU:
+docker run -d -p 5001:5000 my-model
 
-    # If your model uses a GPU:
-    docker run -d -p 5000:5000 --gpus all my-model
+# If your model uses a GPU:
+docker run -d -p 5001:5000 --gpus all my-model
 
-    # If you're on an M1 Mac:
-    docker run -d -p 5000:5000 --platform=linux/amd64 my-model
+# If you're on an M1 Mac:
+docker run -d -p 5001:5000 --platform=linux/amd64 my-model
+```
 
-Port 5000 is now serving the API:
+The server is now running locally on port 5001.
 
-    curl http://localhost:5000
+To view the OpenAPI schema, 
+open [localhost:5001/openapi.json](http://localhost:5001/openapi.json) 
+in your browser 
+or use cURL to make a request:
 
-To run a prediction on the model, call the `/predictions` endpoint, passing input in the format expected by your model:
+```console
+curl http://localhost:5001/openapi.json
```

Comment:
I took the opportunity to run through this doc and test it out. I'm able to see http://localhost:5001/openapi.json just fine, but it looks like port 5000 is not connecting.

```
$ curl http://localhost:5000/predictions -X POST \
    --header "Content-Type: application/json" \
    --data '{"input": "Evazorro"}}'
curl: (7) Failed to connect to localhost port 5000 after 0 ms: Connection refused
```

It may just be user error on my part (or a quirk of using Docker with WSL), but maybe we can debug just to be sure?

Reply:
Fun backstory on this is that [Apple took over port 5000](https://www.reddit.com/r/webdev/comments/qg8yt9/apple_took_over_port_5000_in_the_latest_macos/) by running the AirPlay server that port by default. The first version of Cog rolled out just in time to get bit by that 🥲  

Hence this awkward misdirection where you have to map container 5000 to host 5001.

Reply:
Oof Apple. Ok great, glad I found something.

Reply:
Filed a tracking issue here: #1777